src/components/DeckMechanisms/index.js

```diff
@@ -8,6 +8,17 @@ import resolveDeckWeight, {
   increaseCardWeight,
 } from '../../helpers/resolveDeckWeight'
 
+const DAWNSPARKS_STAYS = 0.71
+const DAWNSPARKS_HIT = 0.71
+const FROZEN_CORE_STAYS = 0.5
+const AHMI_RETURNS = 0.5
+
+export const FRIENDLY_CHANCES = {
+  W9: FROZEN_CORE_STAYS,
+  S3: AHMI_RETURNS,
+  W16: DAWNSPARKS_HIT * DAWNSPARKS_STAYS,
+}
```

Comment:
Very interesting, I like the idea of having this outside of the code itself for better clarity. Maybe we could make this list more explicit like this:

```js
const RNG_DETAILS = {
  W9: {
    FRIENDLY: () => true,
    REGULAR: () => Math.random() > 0.5,
    UNFRIENDLY: () => false
  },
  // …
}
```

Then we could access the chances like so:

```js
const shouldFrozenCoreStay = RNG_DETAILS[card.id][this.state.RNG]()
```

Reply:
I moved the calculations to a helper function - This might no longer be needed